tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -429,13 +430,12 @@ def load_rosparam(self, context, ros_config, cmd, param, file_, text, verbose=Tr
             # 'param' attribute is required for non-dictionary types
             if not param and type(data) != dict:
                 raise ValueError("'param' attribute must be set for non-dictionary values")
-
```

Comment:
Unrelated change. Should be be reverted.

Reply:
Reverted.

Reply:
Reverted.